### PR TITLE
Line spacing

### PR DESCRIPTION
**Pull request by mikesealey**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,11 +118,69 @@ <h2>Mouse Tools</h2>
...
+      <input type="checkbox" id="break-paragraph" data-key="breakParagraph"
+        aria-label="Toggle to break up long paragraphs" />
+      <!-- Label for Background Image Removal toggle -->
+      <label for="break-paragraph">Break up long paragraphs</label>
+      <br />
+    </div>
+
+    <!-- Radio buttons for colour palettes -->
+    <div>
+      <h4>Palette</h4>
```
Comment (ShaneByrne0963) [inline index.html:148]: A lot of duplicate HTML here. I'm guessing you didn't resolve a merge conflict properly. Just copy the raw file from the GitHub page to reset it

File: line-spacing.js [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+      unitsArray.push(character)
+    }
+  })
+
+  let number = Number(numbersArray.join(""))
+  let units = unitsArray.join("")
+
+  element.style.lineHeight = (data.lineSpacing * Number(numbersArray.join(""))) + units
+  element.style.lineSpacing = (data.lineSpacing * Number(numbersArray.join(""))) + units
+  console.log(element.style.lineHeight)
```
Comment (ShaneByrne0963) [inline line-spacing.js:42]: Remove the console.log pls

**Review by ShaneByrne0963 — CHANGES_REQUESTED** [review]
Just need to fix your index.html file and remove the console.log and you're good to go!

File: index.html (hunk 1 above)
Comment (mikesealey) [reply index.html]: `console.log()`s removed, duplicate HTML removed.

**Review by ShaneByrne0963 — APPROVED** [review]
LGTM

**Pull request merged**